packages/angular/src/generators/component-cypress-spec/lib/get-knob-default-value.ts

```diff
@@ -9,7 +9,8 @@ export function getKnobDefaultValue(
   }
   switch (property.initializer.kind) {
     case SyntaxKind.StringLiteral:
-      return property.initializer.getText().slice(1, -1);
+      const returnString = property.initializer.getText().slice(1, -1);
+      return returnString.replace(' ', '+');
```

Comment:
is this doing URL encoding? I wonder whether we can use some URL encoder function? Otherwise this works as well ofc.

Also, would this just replace the first occurrence of a blank space, right? Not sure if that's enough

Reply:
Oh good catch about the first blank space. Yes, I guess I could use the encodeURIcomponent but the thing is I've seen in the generated URL for controls it replaces blank spaces with the `+` sign, that's why I used this.